src/content/cloud/container-resources.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Container Resources
+description: Optimizing Container Resource Allocation for Efficient Development Environments
+sidebar_label: Container Resources
+id: container-resources
+---
+
+import Image from '@theme/Image';
+
+Okteto leverages the concept of Kubernetes resources to decide on which node to place each container.
+When you specify a resource `limit` for a container, Okteto enforces those limits so that the running container is not allowed to use more of that resource than the limit you set.
+Okteto also reserves at least the `request` amount of that system resources specifically for that container to use.
+Follow [this link](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) to know more about how Kubernetes resources work.
+
+## Best Practices
+
+Okteto monitors the CPU and Memory usage of your development environments to balance containers across all the nodes in the cluster.
+However, it's not capable of evicting containers when a node becomes overloaded.
+To avoid this situation, it's strongly recommended that you configure your CPU and Memory `requests` to align with your container's actual resource utilization.
```

Comment:
```suggestion

To avoid this situation, it's strongly recommended that you configure your CPU and Memory `requests` to align with your container's actual resource utilization.
```